RoddyCore/src/de/dkfz/roddy/client/cliclient/RoddyCLIClient.groovy

```diff
@@ -737,26 +737,30 @@ public class RoddyCLIClient {
     /**
      * Sets up Roddys property file from the command line.
      */
-    static void performCommandLineSetup() {
+    static void performCommandLineSetup() throws ConfigurationError {
```

Comment:
I'm not sure whether we should get rid of this method or leave it. Currently the AppConfig class cannot write ini files and the only thing where input is necessary is for the keyword based setup.

Reply:
What do you mean by "keyword-based setup"?

Reply:
If "password" is setup as auth method.